src/tests-rosetta/api-construction.ts

```diff
@@ -551,6 +553,191 @@ describe('Rosetta API', () => {
 
     expect(JSON.parse(result.text)).toEqual(expectedResponse);
   });
+
+  test('parse api signed', async () => {
+    const request: RosettaConstructionParseRequest = {
+      network_identifier: {
+        blockchain: 'stacks',
+        network: 'testnet',
+      },
+      signed: true,
+      transaction:
+        '0x80800000000400164247d6f2b425ac5771423ae6c80c754f7172b0000000000000000000000000000000b400011ae06c14c967f999184ea8a7913125f09ab64004446fca89940f092509124b9e773aef483e925476c78ec58166dcecab3875b8fab8e9aa4213179d164463962803020000000000051a1ae3f911d8f1d46d7416bfbe4b593fd41eac19cb00000000000003e800000000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
Can you change these raw tx payloads to a tx created by the stacks-transactions-js lib? Otherwise it's hard to compare with the below output

Reply:
Yes, that make sense.